map/framework.cpp

```diff
@@ -2178,6 +2178,14 @@ place_page::Info Framework::BuildPlacePageInfo(place_page::BuildInfo const & bui
     // Selection circle should match feature
     FillFeatureInfo(selectedFeature, outInfo);
 
+    if (!outInfo.IsBookmark() && !isBuildingSelected)
+    {
+      // Search for a bookmark at POI position instead of tap position
```

Comment:
We actually search for a bookmark two times during tap handling. Should we check that POI position (`outInfo.GetMercator()`) is far from tap position (`buildInfo.m_mercator`) before performing the second search?